File: Modul 4/soal_5/PRAK405-2410817310003-Muhammad_Guntur_Ricky_Adhitya.c
#include <stdio.h>

int main(){

    int inp_1;
    int inp_2;
    scanf("%d %d", &inp_1, &inp_2);

    int sum = 0;
    for(int i = 0; i < inp_1; ++i){
        int sum_row = 0;
        for(int j = i+1; j >= 1; --j){
            sum_row += j * inp_2;
            printf("(%d * %d)%s", j, inp_2, (j == 1) ? " " : " + ");
        }
        printf("= %d\n", sum_row);
        sum += sum_row;
    }
    printf("%d\n", sum);
    return 0;
}

File: Modul 4/soal_5/PRAK405-2410817310003-Muhammad_Guntur_Ricky_Adhitya.py
inp_1, inp_2 = map(int, input().split())

i = 0
sum = 0
while i < inp_1 :
    sum_row = 0 
    j = i+1
    while j >= 1 :
        sum_row += j * inp_2
        print("(%d * %d)%s"%(j, inp_2, " " if j == 1 else " + "), end="")
        j -= 1
    print("= %d"%sum_row)
    sum += sum_row
    i += 1
print("%d"%sum)